handlers/helpers/helpers.go

```diff
@@ -0,0 +1,31 @@
+package helpers
+
+import (
+	"encoding/json"
+	"github.com/evsyukovmv/taskmanager/logger"
+	"net/http"
+)
+
+func WriteError(w http.ResponseWriter, r *http.Request, err error) {
+	logger.ErrorWithContext(r.Context(), err.Error())
+
+	w.WriteHeader(http.StatusBadRequest)
+	if _, err := w.Write([]byte(`{ error: "` + err.Error() + `" }`)); err != nil {
```

Comment:
use `fmt.Fprintf(w, "format", values...)` instead
https://golang.org/pkg/fmt/#Fprintf

in Go strings are immutable, that's why strings concatenation will cause new memory allocations